frontend/simple/views/Invite.vue

```diff
@@ -133,12 +131,14 @@ export default {
 
       try {
         const contractId = await namespace.lookup(this.searchUser)
+        const state = await latestContractState(contractId)
         if (!this.members.find(member => member.name === this.searchUser)) {
-          this.members.push({ name: this.searchUser, contractId })
+          this.members.push({ name: this.searchUser, picture: state.attributes.picture, contractId })
```

Comment:
And note that you should then delete the no longer necessary call to `latestContractState` on line 155.